mlflow/server/js/src/experiment-tracking/components/CompareRunView.js

```diff
@@ -70,6 +70,61 @@ export class CompareRunView extends Component {
     return (
       <div className='CompareRunView'>
         <PageHeader title={title} breadcrumbs={breadcrumbs} />
+        <h1>
```

Comment:
Make it collapsible after https://github.com/mlflow/mlflow/pull/5306 is merged to avoid duplicate code.